tests/integration_test.py

```diff
@@ -1624,3 +1624,6 @@ def test_649(self):
         ctnr = self.client.create_container('busybox', ['sleep', '2'])
         self.client.start(ctnr)
         self.client.stop(ctnr)
+
+    def test_715(self):
```

Comment:
I'm also a fan of having a descriptive name and linking to the issue in the git commit